MekHQ/src/mekhq/campaign/unit/damage/AeroDamageApplier.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024 - The MegaMek Team. All Rights Reserved.
+ *
+ *  This file is part of MekHQ.
+ *
+ *  MekHQ is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  MekHQ is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with MekHQ. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package mekhq.campaign.unit.damage;
+
+import megamek.common.Aero;
+import megamek.common.HitData;
+import megamek.common.IEntityRemovalConditions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static megamek.common.Compute.randomInt;
+
+/**
+ * Applies damage to an Aero entity.
```

Comment:
Does this apply to larger aero units, like DropShips, WarShips, or JumpShips? If not, how difficult would it be for you to add those handlers too?

Reply:
I've not tried, but it shuold work fine in those. I need to setup a test class to eval it.